### PR TITLE
Feat: added image component

### PR DESCRIPTION
**Pull request by poornima-alugubelly**

### CODE REVIEW

File: components/Image/Image.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Image</title>
+    <link rel="stylesheet" href="./Image.css" />
+  </head>
+  <body>
+    <section style="padding: 1rem" class="container flex-centered">
```
Comment (kshitijthatte) [inline components/Image/Image.html:11]: There is no 'flex-centered' class applied, apart from this everything looks good to me.

**Review by kshitijthatte — APPROVED** [review]

File: components/Image/Image.html (hunk 1 above)
Comment (poornima-alugubelly) [reply components/Image/Image.html:11]: Hey Kshitiji thanks for review yeah I gave the name wrong I think thanks for pointing it out! Its actually not needed and will be removed later anyways tho

**Comment by Swapnilkota108** [comment]
Good to go!

**Review by talrejah02 — APPROVED** [review]

**Review by nshanbhag20001412 — APPROVED** [review]
looks good to me

**Pull request merged**